src/backend/gl/src/command.rs

```diff
@@ -69,6 +69,17 @@ pub enum Command {
     },
     BindIndexBuffer(gl::types::GLuint),
     //BindVertexBuffers(BufferSlice),
+    BindUniform {
+        uniform: n::UniformDesc,
+        buffer: BufferSlice,
+    },
+    BindRasterizer {
+        rasterizer: pso::Rasterizer,
+        is_embedded: bool,
```

Comment:
what's the meaning of this one?

Reply:
The command or the flag? I'm not sure what the flag is for.. I copied it from commented code..